core/src/components/radio-group/radio-group.tsx

```diff
@@ -38,8 +38,6 @@ export class RadioGroup implements ComponentInterface {
   @Watch('value')
   valueChanged(value: any | undefined) {
     this.setRadioTabindex(value);
-
-    this.ionChange.emit({ value });
```

Comment:
I noticed that if you set the `value` of the group programmatically, the selected radio no longer updates. On closer inspection, `ion-radio` is listening for the `ionChange` event in order to handle this: https://github.com/ionic-team/ionic-framework/blob/FW-2033/core/src/components/radio/radio.tsx#L101 We may need to make a separate `ionValueChange` event or something, similar to what `ion-datetime` did.